Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -2238,6 +2238,21 @@
 			);
 			script = "exec \"${SRCROOT}/Scripts/postprocess-header-rule\"\n";
 		};
+		DD3C27842A1E7D2000488D4E /* PBXBuildRule */ = {
+			isa = PBXBuildRule;
+			compilerSpec = com.apple.compilers.proxy.script;
+			dependencyFile = "$(DERIVED_FILES_DIR)/$(INPUT_FILE_PATH).d";
+			filePatterns = "*.sb.in";
+			fileType = pattern.proxy;
+			inputFiles = (
+				"$(SRCROOT)/Scripts/preprocess-sandbox-profile-rule.sh",
+			);
+			isEditable = 1;
+			outputFiles = (
+				"$(BUILT_PRODUCTS_DIR)/DerivedSources/WebKit/$(INPUT_FILE_BASE)",
```

Comment:
`INPUT_FILE_BASE` removes `.in`, but leaves `.sb`?

Reply:
Exactly. It's the file name minus the last extension.